deprecated/frontend/schema/class-schema-article.php

```diff
@@ -41,8 +33,8 @@ class WPSEO_Schema_Article implements WPSEO_Graph_Piece {
 	 */
 	public function __construct( $context = null ) {
 		_deprecated_function( __METHOD__, 'WPSEO 14.0', 'Yoast\WP\SEO\Generators\Schema\Article' );
-		$this->date     = new WPSEO_Date_Helper();
```

Comment:
The `$this->helpers` and `$this->context` properties should still be set.

We may automatically inject them before we call `is_needed` or `generate` ourselves but if someone were to be calling these directly they would now fatal.